book/website/community-handbook/fireside-chat.md

```diff
@@ -0,0 +1,77 @@
+4) Cross-community resource exchange: learning from citizen spaces
+
+### Frequency and Duration
+
+These calls are hosted every month on the third or fourth Friday as per the common availability of all speakers.
+Recommended duration is 60 to 90 minutes that are recorded to post online for people who can't attend these calls in real-time.
+Hosts can consider allocating a part of the call for open discussions with participants, however, this part doesn't need to be recorded to allow maximum participation by attendees who may otherwise not engage.
+The schedule is decided based on the availability of individuals invited as speakers/panellists.
+
+### Resources
+ 
+- **Eventbrite pages**: Registrations are handled via [*The Turing Way* Eventbrite page](https://www.eventbrite.co.uk/o/the-turing-way-18600928389). The event is free to attend and recordings are shared with all registered participants after the event.
+- **Zoom**: We host Fireside events online using [Zoom](https://zoom.us/) (with live transcription), however, the platform for the participants can change based on the preference of the majority of speakers.
+- **Online promotion**: Announcements are shared on *The Turing Way* [Twitter](https://twitter.com/turingway), [Slack workspace](https://tinyurl.com/jointuringwayslack) and [monthly newsletters](https://tinyletter.com/TuringWay), which are then cross-posted by other communities in their network.
+- **Shared Documents**: We currently use Etherpad by Software Freedom Conservancy to enable shared note-taking during the event. See this for example - https://pad.sfconservancy.org/p/ttw-fireside-chat-mar2022. 
+    - **Template** is provided as a subchapter: {ref}`ch-template-fireside-chat`.
+    - **Archived notes** are currently shared via HackMD: https://hackmd.io/@turingway/fireside-chats.
+- **Video Recording**: Video recordings with closed captioning are shared on *The Turing Way* YouTube channel in the [Fireside Chat video playlist](https://www.youtube.com/watch?v=nuNA3Qa8A-k&list=PLBxcQEfGu3DmBcSmt9GVfo4wN1KP1y2yl).
+
+### Participants
+
+- **Hosts**: Two co-hosts, one from The Turing Way team and one from the collaborating project drive the planning and designing of the event, as well as co-chairing the event. 
+- **Speakers**: 3-4 speakers are invited from multiple communities to allow multiple perspectives to be shared. Speakers are invited several weeks before the event to contribute to the event at the planning phase by sharing questions they hope to discuss during the event.
+- **Open participation**: To enable open collaboration during the event, we facilitate open collaboration via Zoom chat and shared note-taking. Although the event is an informal discussion, we record the session upon speakers' permission. We do not run interactive sessions to avoid recording anyone who may not have consented to be recorded. Call hosts can plan to allocate part of the event for open discussions that should not be recorded.
```

Comment:
would suggest "via Zoom chat and shared note-taking on Etherpad".

Reply:
```suggestion
- **Open participation**: To enable open collaboration during the event, we facilitate open collaboration via Zoom chat and shared note-taking in Etherpad. Although the event is an informal discussion, we record the session upon speakers' permission. We do not run interactive sessions to avoid recording anyone who may not have consented to be recorded. Call hosts can plan to allocate part of the event for open discussions that should not be recorded.
```